<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>segyanalyzer</class>
 <widget class="QWidget" name="segyanalyzer">
  <property name="windowModality">
   <enum>Qt::NonModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1342</width>
    <height>725</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>SEGY Analyzer</string>
  </property>
  <property name="windowIcon">
   <iconset resource="resources.qrc">
    <normaloff>:/img/icon.ico</normaloff>:/img/icon.ico</iconset>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <property name="leftMargin">
      <number>9</number>
     </property>
     <property name="rightMargin">
      <number>9</number>
     </property>
     <property name="bottomMargin">
      <number>6</number>
     </property>
     <item>
      <widget class="QPushButton" name="btnSelectSEGYFiles">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>50</height>
        </size>
       </property>
       <property name="font">
        <font>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>Select SEGY Files   </string>
       </property>
       <property name="icon">
        <iconset resource="resources.qrc">
         <normaloff>:/img/files.png</normaloff>:/img/files.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>32</width>
         <height>32</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnSetByteLocation">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>50</height>
        </size>
       </property>
       <property name="font">
        <font>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>Set Byte Location   </string>
       </property>
       <property name="icon">
        <iconset resource="resources.qrc">
         <normaloff>:/img/byte.png</normaloff>:/img/byte.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>32</width>
         <height>32</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnStartAnalyze">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>50</height>
        </size>
       </property>
       <property name="font">
        <font>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>Start Analyze !   </string>
       </property>
       <property name="icon">
        <iconset resource="resources.qrc">
         <normaloff>:/img/start.png</normaloff>:/img/start.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>32</width>
         <height>32</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>178</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="btnPrint">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>50</height>
        </size>
       </property>
       <property name="text">
        <string>Print</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="Line" name="line">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <property name="leftMargin">
      <number>9</number>
     </property>
     <property name="rightMargin">
      <number>9</number>
     </property>
     <item>
      <widget class="QLabel" name="lbDirPath">
       <property name="text">
        <string>Directory: -</string>
       </property>
       <property name="margin">
        <number>3</number>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_3">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QLabel" name="lbByteLocation">
       <property name="text">
        <string>CDP=&lt;strong&gt;1&lt;/strong&gt; , SP=&lt;strong&gt;17&lt;/strong&gt; , X=&lt;strong&gt;73&lt;/strong&gt; , Y=&lt;strong&gt;77&lt;/strong&gt;</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>208</width>
         <height>18</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QLabel" name="lbTotalFiles">
       <property name="text">
        <string>Total Files: -</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <property name="leftMargin">
      <number>9</number>
     </property>
     <property name="rightMargin">
      <number>9</number>
     </property>
     <item>
      <widget class="QTableWidget" name="tableSEGY">
       <property name="editTriggers">
        <set>QAbstractItemView::NoEditTriggers</set>
       </property>
       <property name="dragDropOverwriteMode">
        <bool>false</bool>
       </property>
       <property name="alternatingRowColors">
        <bool>true</bool>
       </property>
       <property name="selectionMode">
        <enum>QAbstractItemView::SingleSelection</enum>
       </property>
       <property name="selectionBehavior">
        <enum>QAbstractItemView::SelectRows</enum>
       </property>
       <property name="gridStyle">
        <enum>Qt::SolidLine</enum>
       </property>
       <attribute name="horizontalHeaderCascadingSectionResizes">
        <bool>false</bool>
       </attribute>
       <attribute name="horizontalHeaderStretchLastSection">
        <bool>false</bool>
       </attribute>
       <attribute name="verticalHeaderStretchLastSection">
        <bool>false</bool>
       </attribute>
       <column>
        <property name="text">
         <string>Filename</string>
        </property>
        <property name="background">
         <color>
          <red>0</red>
          <green>170</green>
          <blue>255</blue>
         </color>
        </property>
       </column>
       <column>
        <property name="text">
         <string>Line Name</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>Total Traces</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>Sample Rate</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>End Time</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>First CDP</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>Last CDP</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>CDP Inc.</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>First SP</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>Last SP</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>SP Inc.</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>First X</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>Last X</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>First Y</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>Last Y</string>
        </property>
       </column>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections/>
</ui>
